packages/plugin-dom-layout/src/DomReconciliationEngine.ts

```diff
@@ -212,8 +235,22 @@ export class DomReconciliationEngine {
         const allOldDomNodes: Node[] = [];
         for (const id of removeObjects) {
             const old = this._objects[id];
+            if (!old) {
+                console.warn(
+                    'A rendering object are remove twice, one of the references was wrong.',
+                );
+                continue;
+            }
             if (typeof old.object.detach === 'function') {
-                old.object.detach(...old.dom);
+                try {
+                    old.object.detach(...old.dom);
+                } catch (e) {
+                    console.warn(
+                        'Can not detach the nodes from a rendering object, one of the references was may be wrong.',
```

Comment:
Une erreur est apparue lors du detach. Il se peut que le rendu cible aie déjà été effacé. Il existe donc plus que une référence vers cet élément, et donc il y a au moins une reférence erronnée.

Reply:
An error occured while detaching. The target rendering might already have been deleted. Multiple references might target this rendering, in which case at least one of them is wrong.